arch/risc-v/src/c906/Make.defs

```diff
@@ -0,0 +1,61 @@
+############################################################################
+# arch/risc-v/src/c906/Make.defs
+#
+#   Copyright (C) 2019 Masayuki Ishikawa. All rights reserved.
```

Comment:
@masayuki2009 has already filed a ICLA so we can drop this from the license header.